mos_tests/environment/fuel_client.py

```diff
@@ -48,9 +48,10 @@ def get_ssh_to_node(self, ip):
             private_keys=self.admin_ssh_keys
         )
 
-    def get_ssh_to_cirros(self, ip, private_keys):
+    def get_ssh_to_cirros(self, ip, private_keys=None,
+                          creds=('cirros', 'cubswin:)')):
         return SSHClient(
-            host=ip, username="cirros", password=None,
+            host=ip, username=creds[0], password=creds[1],
```

Comment:
Can I do it like that? 
def get_ssh_to_cirros(self, ip, username='cirros', password='cubswin:)',
                                   private_keys=None):